release/github/publish.sh

```diff
@@ -59,9 +66,9 @@ compile() {
 }
 
 # Now compiler ibazel for every platform/arch that is supported.
-compile "linux"   "amd64"
-compile "darwin"  "amd64"
-compile "windows" "amd64"
+compile "linux"   "amd64"  ""
+compile "darwin"  "amd64"  "cgo"
```

Comment:
@achew22, I do not know exactly where this is built and am not super familiar with how rules_go works with cgo. I imagine this might cause issues on where its built... Any thoughts here?

Reply:
Is this actually compiling and creating new binaries in the correct output dirs? If that's the case then I think the best course is going to be for me to clone it locally and use a disassembler or something to look at how the files are actually assembled.

TL;DR: I have low confidence in the flags doing things I expect

Reply:
It is still compiling 3 binaries, one for each platform. I changed the bash function to accept a 3rd argument and change namings based on it. 

What I was worried about is being able to cross-compile to macOS from linux now that cgo is on. I do not know if that is possible anymore, but I *think* its a requirement to use FSEvents since its an OS framework.